test/recursive_denormalized_has_many_test.rb

```diff
@@ -61,7 +61,7 @@ def test_on_cache_miss_child_record_fetch_should_include_nested_associations_to_
       # one for the mid level has_one association
       # one for the deep level level has_many on the mid level has_many association
       # one for the deep level level has_many on the mid level has_one association
-      record_from_cache_miss = Item.fetch(@record.id)
+      Item.fetch(@record.id) # record_from_cache_miss
```

Comment:
I think `# cache miss` would be a more appropriate comment.  Although I don't think a cache miss comment is necessary since the test name makes it clear that it is testing a cache miss.